### ISSUE TITLE
[Update Request]: KDE.KDEConnect URL invalid (404)

### ISSUE THREAD

**Issue by jkwek** [microsoft/winget-pkgs#122904]
### What type of update are you requesting?

A change to the package metadata

### Current Package Identifier

KDE.KDEConnect

### Package Version

23.08.1

### Please describe the changes you would like to see

Getting this when trying to install KDEConnect:

> winget install -e --id KDE.KDEConnect
Found KDE Connect [KDE.KDEConnect] Version 23.08.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://binary-factory.kde.org/view/Windows%2064-bit/job/kdeconnect-kde_Release_win64/1470/artifact/kdeconnect-kde-23.08.1-1470-windows-cl-msvc2019-x86_64.exe
An unexpected error occurred while executing the command:
Download request status is not success.
0x80190194 : Not found (404).

Downloading manifest files and changing Installer URL to  https://download.kde.org/stable/release-service/23.08.1/windows/kdeconnect-kde-23.08.1-1463-windows-cl-msvc2019-x86_64.exe and updating the InstallerSha256 results in a successful install.  Recommending to change the URL and update the checksum and use the download.kde.org URL going forward for new releases.

**Comment by jkwek** [comment]
Works great, thank you!  I would close this out, but wanted to check if this will be fixed going forward? I think download.kde.org should be the default location going forward as binary-factory.kde.org seems to be more temporary and files don't stick around there for too long.

**Comment by shivaabhishek07** [comment]
Yeah for sure, let's make use of download.kde.org for further versions.

**Comment by jkwek** [comment]
Great!  I will close this out.  Thank you for fixing this!

**Comment by jkwek** [comment]
This change has not applied to newer versions of KDE Connect.  Versions 23.08.3 and 23.08.4 still have the bad URL.  I'm reopening so this can be fixed for future versions.  Commits have been from @vedantmgoyal2009 in case this needs to be raised.
